### DISCUSSION TITLE
`.webp` support for `.Media()` blocks

### DISCUSSION THREAD

**Discussion by teadetime** [datapane/datapane#363] [Ideas]
### Goals

1. Add support for `.webp` images in `Media` blocks

### Non-Goals

1.
2.
3.

### Background

Enables larger Datapanes! I'm on a project that has ~30mb page using Plotly graphs. We want to save Plotly graphs to `.webp` which is has the smallest file size/quality. This allows for pages that are of a staggering size with Plotly (on the order of displaying 1000's or 10,000s of graphs via images). 

### Proposal

I did a quick look into the media blocks. It seems like there needs to be a little work in `notebook.ts` to support an `images/webp` type. I imagine there are other requirements and changes elsewhere. 

I also anticipate there is a larger discussion or a prior decision not to support `.webp` as it is not supported on all browsers ([just IE](https://caniuse.com/webp) at this point I think). 

I couldn't find an architecture overview to understand if I was getting the whole picture, but would happy to try implementing this if someone could provide a little more direction!

**Comment by mands** [comment]
Hi there @teadetime - this should be pretty easy for us to pick up using the existing media framework.

Pinging @Buroni who is more familiar with the renderer and will be able to dig into it a bit further.
